.github/workflows/ci-linux.yml

```diff
@@ -9,7 +9,7 @@ on:
       - '**.txt'
 
 env:
-  alire_index: ""
+  alire_index: "git+https://github.com/alire-project/alire-index@devel-0.3"
```

Comment:
This should not be necessary because the index is determined by `Alire.Index.Community_Branch`, so this will result in extra maintenance for no gain. In fact, it's a remainder of when we could not select an index in any other way. 

So, is it giving you some trouble that I'm missing?

Reply:
Actually I made several attempts to get the testsuite to run, so this might not be necessary in the end.